addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiDeviceBaseHandler.java

```diff
@@ -107,7 +119,9 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void onItemUpdate(String sid, String command, JsonObject message) {
         if (getItemId() != null && getItemId().equals(sid)) {
-            updateThingStatus();
+            if (getThing().getStatus() != ThingStatus.ONLINE) {
```

Comment:
Is this check really need, will something fail, afaik it will not trigger or log anything if you update it to the same value, @htreu wdyt?

Reply:
From what I know you are right: telling the framework about the same status multiple times does not do any harm.